frontend/packages/schema/src/parser/drizzle/postgres/__tests__/index.test.ts

```diff
@@ -57,5 +57,64 @@ describe(processor, () => {
         'numeric(10,2)',
       )
     })
+
+    it('should parse tables with .enableRLS() method chaining', async () => {
+      const { value } = await processor(`
+        import { pgTable, serial, text, varchar } from 'drizzle-orm/pg-core';
+
+        export const users = pgTable('users', {
+          id: serial('id').primaryKey(),
+          name: varchar('name', { length: 255 }).notNull(),
+          email: text('email').notNull(),
+        }).enableRLS();
+      `)
+
+      // Verify that table with .enableRLS() is correctly parsed
+      expect(Object.keys(value.tables)).toHaveLength(1)
+      expect(value.tables['users']).toBeDefined()
+
+      // Verify table structure
+      expect(value.tables['users']?.columns['id']?.type).toBe('serial')
+      expect(value.tables['users']?.columns['name']?.type).toBe('varchar(255)')
+      expect(value.tables['users']?.columns['email']?.type).toBe('text')
+    })
+
+    it('should parse tables with .$comment() method chaining', async () => {
+      const { value } = await processor(`
+        import { pgTable, serial, text, varchar } from 'drizzle-orm/pg-core';
+
+        export const users = pgTable('users', {
+          id: serial('id').primaryKey(),
+          name: varchar('name', { length: 255 }).notNull(),
+          email: text('email').notNull(),
+        }).$comment('User accounts table');
```

Comment:
Thank you for not only implementing `.enableRLS()` but also investigating Drizzle's syntax and making a universal fix! Great work!